src/gateway/understanding-kong/key-concepts/services.md

```diff
@@ -1,6 +1,33 @@
 ---
 title: Services
-
+content_type: explanation
 ---
 
-## PLACEHOLDER Services
+In {{site.base_gateway}}, a service is an entity representing an external upstream API or microservice. For example, a data transformation microservice, a billing API, and so on. 
+
+The main attribute of a service is its URL, where Kong should proxy traffic to. You can specify the URL with a single string, or by specifying its protocol, host, port, and path individually. 
+
+Service entities are abstractions of each of your own upstream services. Examples of Services would be a data transformation microservice, a billing API, etc.
+
+## Service configuration
+
+When configuring access to your API, you start by specifying a service.
+
+You can configure a service in {{site.base_gateway}} using the following methods:
+
+* Send an HTTP request using the Admin API
+* Create a service using the Kong Manager user interface
+
+## Service and route interaction
+
+Services, in conjunction with [routes](/gateway/latest/understanding-kong/key-concepts/routes/), let you expose your services to clients with {{site.base_gateway}}. {{site.base_gateway}} abstracts the service from the clients by using routes. Since the client always calls the route, changes to the services(like versioning) don't impact how clients make the call.  Routes also allow the same service to be used by multiple clients and apply different policies based on the route used. 
```

Comment:
```suggestion
Services, in conjunction with [routes](/gateway/latest/understanding-kong/key-concepts/routes/), let you expose your services to clients with {{site.base_gateway}}. {{site.base_gateway}} abstracts the service from the clients by using routes. Since the client always calls the route, changes to the services(like versioning) don't impact how clients make the call. Routes also allow the same service to be used by multiple clients and apply different policies based on the route used. 
```